test/test-server/ccloudv1_handlers.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 // Handler for "/api/organizations/"
-func (c *CloudRouter) HandlePriceTable(t *testing.T) http.HandlerFunc {
+func handlePriceTable(t *testing.T) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
```

Comment:
some handlers don't have the router receiver? How is `router.HandleFunc(route.path, route.handler(t))` working with them?

Reply:
I'm iterating through a list of `route`s, which have `path` and `handler` fields. (Note the difference between `route` and `router`)